### ISSUE TITLE
same autoencoder trains very slow if component of Autoencoding UMAP - n_epoch and n_training_epochs in parametric UMAP

### ISSUE THREAD

**Issue by gg4u** [lmcinnes/umap#1036]
not clear to me the difference between `n_epoch` and `n_training_epochs`, and I see a huge difference in training  time, between:
- training an Autoencoder model separately, for 100 epochs, than project the embedding with non-parametric UMAP (1secon /epoch)
- training the same autoencoder as part in the Parametric UMAP, for 10 epochs (5 minute / epochs)
They are more are one order of magnitude in the training time, and don't understand when in the paper you say:

```
 Across
each dataset, we find that non-parametric UMAP reaches a lower loss more quickly
than Parametric UMAP, but that Parametric UMAP reaches a similar cross-entropy
within an order of magnitude of time. Thus, Parametric UMAP can train more slowly
than non-parametric UMAP, but training times remain within a similar range making
Parametric UMAP a reasonable alternative to non-parametric UMAP in terms of training
time
```

https://arxiv.org/pdf/2009.12981.pdf

- can you clarify if your findings is related to the choice of BinaryCrossEntropy ? I am using MSE as loss? 
- can you clarify the difference of the parameters `n_epoch` and `n_training_epochs` ?
In training a normal tensorflow models with epochs  = 100, it trIsiain 100 times.
In training autoencoding umap with n_training_epochs = 1 , it trains 10 times (10 epochs). 
I see Parametric UMAp is also learning the projection embedding, but not clear what the epochs in Parametric Umap actually do VS normal TF model.
